evalml/pipelines/components/transformers/preprocessing/time_series_featurizer.py

```diff
@@ -190,9 +190,11 @@ def _compute_rolling_transforms(self, X, y, original_features):
             min_periods=size + 1,
         )
         rolling_mean = rolling_mean.get_function()
-        numerics = set(
-            X.ww.select(["numeric"], return_schema=True).columns
-        ).intersection(original_features)
+        numerics = sorted(
```

Comment:
I believe the test passes without this change. @bchen1116 May need to add some more numeric columns to the test case to test the intended behavior.